tools/nut-scanner/nut-scanner.c

```diff
@@ -210,6 +219,7 @@ void show_usage()
 	}
 
 	printf("  -E, --eaton_serial <serial ports list>: Scan serial Eaton devices (XCP, SHUT and Q1).\n");
+        printf("  -T, --thread <max number of threads>: max number of simultaneaous threads (default %d). \n", DEFAULT_THREAD);
```

Comment:
The -T option is a nop if pthread support is missing, maybe not advertise it in such case?